pkg/builder/controller.go

```diff
@@ -123,10 +129,54 @@ type WatchesInput struct {
 	objectProjection objectProjection
 }
 
-// Watches exposes the lower-level ControllerManagedBy Watches functions through the builder.  Consider using
-// Owns or For instead of Watches directly.
+// Watches defines the type of Object to watch, and configures the ControllerManagedBy to respond to create / delete /
+// update events by *reconciling the object* with the given EventHandler.
+//
+// This is the equivalent of calling
+// Watches(source.Kind(scheme, object), eventhandler, opts...).
```

Comment:
That one is also wrong. Ideally grep for Watches, not sure if there are more

Reply:
Fixed

Reply:
Thx!